poetry/console/commands/export.py

```diff
@@ -46,6 +49,7 @@ def handle(self):
         exporter.export(
```

Comment:
I think we can pass the command's io to the exporter. This will help testing the write to stdout without relying to the actual stdout.